pkg/crypto/crypto.go

```diff
@@ -61,17 +64,33 @@ func encrypt(plaintext string, key []byte) (string, error) {
 	writer.Close()
 	encoder.Close()
 
-	return string(msgBuffer.Bytes()), nil
+	// Remove unnecessary characters and trim new lines before storing as
+	// part of node Annotations username.
+	encryptedStr := string(msgBuffer.Bytes())
+	encryptedStr = strings.Replace(encryptedStr, startTag, "", -1)
+	encryptedStr = strings.Replace(encryptedStr, endTag, "", -1)
+	encryptedStr = strings.Trim(encryptedStr, "\n")
```

Comment:
Can't `\n` also be part of an encrypted string i.e. not just near the start and end tags?

Reply:
It can, but we replace it with wmc_marker so that when we decrypt we can replace them back with `\n`